From b48d6f0d45ad90fb4af0f3598cc96d249fa47a56 Mon Sep 17 00:00:00 2001
From: Mark Lee <gmusicapi@lazymalevolence.com>
Date: Sun, 29 May 2016 16:17:48 -0700
Subject: [PATCH] Use pycryptodomex instead of pycryptodome

This prevents weird conflicts when other dependencies use PyCrypto
features that have been renamed (or otherwise) in PyCryptodome.
---
 gpsoauth/google.py | 4 ++--
 requirements.txt   | 2 +-
 setup.py           | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/gpsoauth/google.py b/gpsoauth/google.py
index bbb7728..4901e16 100644
--- a/gpsoauth/google.py
+++ b/gpsoauth/google.py
@@ -1,8 +1,8 @@
 import base64
 import hashlib
 
-from Crypto.PublicKey import RSA
-from Crypto.Cipher import PKCS1_OAEP
+from Cryptodome.PublicKey import RSA
+from Cryptodome.Cipher import PKCS1_OAEP
 
 from .util import bytes_to_long, long_to_bytes
 
diff --git a/requirements.txt b/requirements.txt
index 6b59110..140855e 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,2 +1,2 @@
-pycryptodome==3.4
+pycryptodomex==3.4
 requests==2.9.1
diff --git a/setup.py b/setup.py
index 93f1844..00cae0a 100644
--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,7 @@
     packages=['gpsoauth'],
     include_package_data=True,
     install_requires=[
-        'pycryptodome >= 3.0',
+        'pycryptodomex >= 3.0',
         'requests',
     ],
     license='MIT',
